sync3/caches/global.go

```diff
@@ -72,22 +73,24 @@ func NewGlobalCache(store *state.Storage) *GlobalCache {
 	}
 }
 
-func (c *GlobalCache) OnRegistered(_ int64) error {
+func (c *GlobalCache) OnRegistered(_ context.Context, _ int64) error {
 	return nil
 }
 
 // Load the current room metadata for the given room IDs. Races unless you call this in a dispatcher loop.
 // Always returns copies of the room metadata so ownership can be passed to other threads.
 // Keeps the ordering of the room IDs given.
-func (c *GlobalCache) LoadRooms(roomIDs ...string) map[string]*internal.RoomMetadata {
+func (c *GlobalCache) LoadRooms(ctx context.Context, roomIDs ...string) map[string]*internal.RoomMetadata {
 	c.roomIDToMetadataMu.RLock()
 	defer c.roomIDToMetadataMu.RUnlock()
 	result := make(map[string]*internal.RoomMetadata, len(roomIDs))
 	for i := range roomIDs {
 		roomID := roomIDs[i]
 		sr := c.roomIDToMetadata[roomID]
 		if sr == nil {
-			logger.Error().Str("room", roomID).Msg("GlobalCache.LoadRoom: no metadata for this room")
+			const errMsg = "GlobalCache.LoadRoom: no metadata for this room"
+			logger.Error().Str("room", roomID).Msg(errMsg)
+			internal.GetSentryHubFromContextOrDefault(ctx).CaptureException(fmt.Errorf(errMsg))
```

Comment:
This isn't sentry worthy, it can happen when rooms are initially loaded into the cache.

Reply:
Okay. Shall we downgrade to a warning then?